### ISSUE TITLE
[Supporting the platform – Make Code - Project Setting] Toggle buttons are getting disappeared after applying high contrast (Black/White) theme under 'Project Setting' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3685]
**User Experience**:
Visual impaired user will find difficulty to see the control or element if that control does not adopt the high contrast mode, as they won't be able to take on that controls and they will miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Project Settings" option and select it.
7. Navigate though all the controls appearing after select "Project Setting" option i.e. No Pairing Required: Anyone can connect via Bluetooth, Passkey pairing: Pairing requires 6 digit key to pair etc. 
8. Verify that toggle buttons are getting disappeared after applying high contrast (Black/White) theme or not.

**Actual Result:**
Toggle buttons are getting disappeared after applying high contrast (Black/White) theme under 'Project Setting' screen.

**Expected Result:**
Toggle buttons should not get disappear after applying high contrast (Black/White) theme under 'Project Setting' screen. Toggle buttons should be clearly visible in high contrast mode.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev1; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.3.1; #Win10-Edge(Chromium)

![WCAG4 3 1_Project Setting_HC](https://user-images.githubusercontent.com/72871089/99031412-06da6000-259d-11eb-802c-4ceb8a0c44d1.jpg)
